flutter_local_notifications_windows/lib/src/details/notification_to_xml.dart

```diff
@@ -0,0 +1,39 @@
+import "package:xml/xml.dart";
+
+import "notification_details.dart";
+
+/// Converts a notification with [WindowsNotificationDetails] into well-formed XML.
+///
+/// For more details, refer to the [Toast Notification XML schema](https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/schema-root).
+String notificationToXml({
```

Comment:
The entire file is being exported and means `notificationToXml()` is exposed by the plugin and suspect it's better off to not to do so unless I missed why this end for

Reply:
Fixed